### DISCUSSION TITLE
请问mac-开发模式下，刷新浏览器后头2个请求会很慢，是否是因为刷新浏览器，代理在重新启动？

### DISCUSSION THREAD

**Discussion by zhouzyc** [umijs/umi#9852] [Q&A]
1.win目前没有这个情况，只有mac下会这样。
2.如果是正常登录进入，请求就正常，如果是刷新当前页面，会出现等待。
3.每次刷新浏览器感觉等待时间很长。
![WX20221124-143149@2x](https://user-images.githubusercontent.com/18420689/203711358-ebbd0941-7cd5-4e84-a426-bf571e06decd.png)
![WX20221124-143136@2x](https://user-images.githubusercontent.com/18420689/203711375-ff07272c-df0f-4bd6-93fa-341c10888edc.png)

-》再次刷新
![WX20221125-105713@2x](https://user-images.githubusercontent.com/18420689/203891503-d162da95-6c0f-4500-a821-524de69db856.png)
![WX20221125-105744@2x](https://user-images.githubusercontent.com/18420689/203891519-fe65bd30-dfa0-4b6b-9549-33285ee38737.png)
这个是整个页面加载流。最后一个请求还超时了，加载图片和ico也显示时间长，刷新页面才会出现。
1.会不会mac上的"express"框架有不同？
2.刷新浏览器umi重新启动了代理？或者是什么比较耗时的？
3.代理没有连接上？被阻塞了？

要懵逼了都，用其它框架比如taro就不会出现这样情况，不晓得从什么地方入手来查一下

**Comment by fz6m** [comment]
多给点上下文信息帮助排查问题，配置，电脑信息，代码请求是什么样的等。

**Reply by zhouzyc** [reply]
> 多给点上下文信息帮助排查问题，配置，电脑信息，代码请求是什么样的等。

电脑：macOS Catalina 10.15.4 (19E287)
umi4+node16.14.0+npm8.3.1.

**Reply by fz6m** [reply]
配置，最小复现。
